src/mongo/db/audit/audit_options_test.cpp

```diff
@@ -0,0 +1,220 @@
+/**
+ *    Copyright (C) 2018-present MongoDB, Inc.
```

Comment:
Use Percona license for the new files. For example of Percona license see `src/mongo/db/encryption/kmip_client.cpp`. Change year in the license to the current one (2025-present).

Reply:
Fixed.